src/smart-components/myUserAccess/MUAAccessTable.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState }  from 'react';
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+import { getPrincipalAccess } from '../../redux/actions/access-actions';
+import { defaultSettings } from '../../helpers/shared/pagination';
+import { TableToolbarView } from '../../presentational-components/shared/table-toolbar-view';
+import { createRows } from './mua-table-helpers';
+
+const columns = [
+  { title: 'Application' },
+  { title: 'Resource type' },
+  { title: 'Operation' }
+];
+
+const MUAAccessTable = () => {
+
+  const dispatch = useDispatch();
+  const { permissions, isLoading } = useSelector(state => ({
+    permissions: state.accessReducer.access,
+    isLoading: state.accessReducer.isLoading
+  }), shallowEqual);
+
+  const [ config, setConfig ] = useState({
```

Comment:
It's better to destruct in here
```suggestion
  const [ { pagination, filter }, setConfig ] = useState({
```